nailgun/entities.py

```diff
@@ -1205,7 +1205,7 @@ def __init__(self, server_config=None, **kwargs):
         super(ContentUpload, self).__init__(server_config, **kwargs)
 
 
-class ContentViewVersion(Entity, EntityReadMixin, EntityDeleteMixin):
+class ContentViewVersion(Entity, EntityReadMixin, EntityDeleteMixin, EntitySearchMixin):
```

Comment:
Line is too long
`/nailgun/entities.py:1208:80: E501 line too long (88 > 79 characters)`

we keep a limit to 79 chars.

```python
class ContentViewVersion(
        Entity, 
        EntityReadMixin, 
        EntityDeleteMixin, 
        EntitySearchMixin):
    """A representation of a Content View Version non-entity."""
    def __init__(self, server_config=None, **kwargs):
```